core/src/main/java/tc/oc/pgm/command/StatsCommand.java

```diff
@@ -21,21 +21,23 @@
       aliases = {"stats"},
       desc = "Show your stats for the match")
   public void stats(Audience audience, CommandSender sender, MatchPlayer player, Match match) {
+    if (player.getSettings().getValue(SettingKey.STATS).equals(SettingValue.STATS_ON)) {
+      throw TextException.of("match.stats.disabled");
```

Comment:
Why not just let them see their stats? My thought was if they disable stats it won't automatically display at the end of the match, but if they really want to see it, by all means let them.

Reply:
That makes sense, ill remove this code from the command